### PR TITLE
✨ PRTL-1138 link to expo page on #cases in project chart

### PR DESCRIPTION
**Pull request by cheapsteak**
image

The link will result in 0 cases/mutations/genes due to missing data

The code AFAIK is working as expected

### CODE REVIEW

File: modules/node_modules/@ncigdc/components/ProjectsCharts.js [hunk 1]
```diff
@@ -5,9 +5,10 @@ import Measure from 'react-measure';
 import Relay from 'react-relay/classic';
 import QuestionIcon from 'react-icons/lib/fa/question-circle';
 import * as d3 from 'd3';
-import { compose, withState, withProps, lifecycle, mapProps } from 'recompose';
+import { compose, withState, withProps, lifecycle, mapProps, withPropsOnChange } from 'recompose';
```
Comment (Jephuff) [inline modules/node_modules/@ncigdc/components/ProjectsCharts.js]: This isn't used anymore is it?

File: modules/node_modules/@ncigdc/components/ProjectsCharts.js [hunk 2]
```diff
@@ -17,16 +18,19 @@ import { Tooltip } from '@ncigdc/uikit/Tooltip';
 
 import withRouter from '@ncigdc/utils/withRouter';
 import { fetchApi } from '@ncigdc/utils/ajax';
-import { setFilter, mergeQuery } from '@ncigdc/utils/filters';
+import { setFilter, mergeQuery, makeFilter } from '@ncigdc/utils/filters';
```
Comment (Jephuff) [inline modules/node_modules/@ncigdc/components/ProjectsCharts.js]: this also isn't used?

File: modules/node_modules/@ncigdc/components/ProjectsCharts.js [hunk 3]
```diff
@@ -305,8 +302,14 @@ const ProjectsChartsComponent = compose(
         </div>
         {!genesIsFetching ? [
           <div style={{ alignSelf: 'center', color: theme.greyScale7, fontSize: '1.2rem' }} key='bar-subtitle'>
+            <ExploreLink
+              query={{
+                searchTableTab: 'cases',
+                filters: { op: 'and', content: filters },
```
Comment (Jephuff) [inline modules/node_modules/@ncigdc/components/ProjectsCharts.js]: filters should either be `null` or the full filter object so you should be able to just do `{ filters }`.
shouldn't this also use `massagedFilters`?

**Comment by cheapsteak** [comment]
closing in favor of https://github.com/NCI-GDC/portal-ui/pull/1679

**Pull request closed without merging**